libs/shared/src/types/framework/discover.types.ts

```diff
@@ -1,10 +1,10 @@
-import { ActionStepEnum, ChannelStepEnum } from '../constants';
 import { JsonSchema, Schema } from './schema.types';
 import { StepOptions } from './step.types';
 import { Execute, WorkflowOptions } from './workflow.types';
 import { Awaitable } from './util.types';
+import { ActionTypeEnum, ChannelTypeEnum } from '../channel';
 
-export type StepType = `${ChannelStepEnum | ActionStepEnum}`;
+export type StepTypeString = `${ChannelTypeEnum | ActionTypeEnum}`;
```

Comment:
we had duplication of type `StepType` which were:
1. StepType = ChannelTypeEnum | ActionTypeEnum;
2. StepType = `${ChannelTypeEnum | ActionTypeEnum}`;

As a temporary solution in order to not break the interface i kept both and renamed the template literal (the second StepType) to StepTypeString, i would like to hear your opinion on this matter.